shared/common-adapters/icon.paths.native.js

```diff
@@ -2,767 +2,44 @@
 // @flow
```

Comment:
Can we switch the order of this? `// @flow` should be at the top, otherwise flow doesn't actually check this file (I know)